components/sidebar/sidebar_header/sidebar_header.tsx

```diff
@@ -131,9 +132,15 @@ const SidebarHeader: React.FC<Props> = (props: Props): JSX.Element => {
                         onToggle={handleMenuToggle}
                         className='SidebarHeaderMenuWrapper test-team-header'
                     >
-                        <SidebarHeading>
-                            <span className='title'>{currentTeam.display_name}</span>
-                            <i className='icon icon-chevron-down'/>
+                        <SidebarHeading >
```

Comment:
Would it be better practice to add a button as a child inside here instead of using tabIndex?
Something like this:
```
<SidebarHeading>
     <button className='style--none'>
           <span className='title'>{currentTeam.display_name}</span>
           <i className='icon icon-chevron-down'/>
     </button>
</SidebarHeading>
```
Note the button also needs `display:flex`.

This makes the whole header one tabbable object instead of going through the text and then the icon. In the current fix we are tabbing to the team header which doesn't do any action and then to the icon